spec/game_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'game'
+
+describe Game do
+  
+  it 'calculates a gutter game' do
```

Comment:
This is specifically describing the results of the "points" method, therefore I think you should incude this in another description block, e.g.
`describe "#points" do`
`it "calculates a gutter game"`